src/integrationTest/resources/application.properties

```diff
@@ -30,6 +30,9 @@ scheduling.task.delete-rejected-files.enabled=false
 scheduling.task.reject-duplicates.enabled=false
 scheduling.task.reject-duplicates.cron=0/10 * * * * *
 
+scheduling.task.send-notifications.enabled=false
+scheduling.task.send-notifications.cron=0/10 * * * * *
```

Comment:
is this property needed if the job is disabled ?

Reply:
Just followed the config added for existing tasks. Removed the cron property now